File: 05. Percabangan/E. If Then Case/e_if_then_case.py
N = int(input())
if (N > 9999):
    print("puluhribuan")
elif (N > 999):
    print("ribuan")
elif (N > 99):
    print("ratusan")
elif (N > 9):
    print("puluhan")
else:
    print("satuan")

File: 05. Percabangan/E. If Then Case/e_if_then_case.rs
fn main() {
    let mut reader = String::new();
    std::io::stdin().read_line(&mut reader).expect("");
    let n: i32 = reader.trim().parse().expect("");
    if n > 9999{
        println!("puluhribuan");
    }
    else if n > 999{
        println!("ribuan");
    }
    else if n > 99{
        println!("ratusan");
    }
    else if n > 9{
        println!("puluhan");
    }
    else{
        println!("satuan");
    }
}
